changelogs/head.asciidoc

```diff
@@ -21,3 +21,9 @@ https://github.com/elastic/apm-server/compare/8.9\...main[View commits]
 - Add a self-instrumentation transaction to the agent config periodic refresh action. {pull}11129[11129]
 - Stop dropping metadata fields from span documents. {pull}11089[11089]
 - Add permissions to reroute events in the integration package. {pull}11168[11168]
+
+[float]
+==== Aggregation improvements
+- Replace aggregation implementation {pull}11117[11117]
+- Add `service.language.name` to service destination metrics {pull}11117[11117]
+- Modify per-service transaction groups limit to consider more than service.name; Add per-service service destination groups limit and per-service service transaction groups limit {pull}11117[11117]
```

Comment:
@simitt WDYT about the changelog?

Reply:
my 2c: I think we should improve the first message to briefly mention what it means for the user. At the very least we should mention the significantly reduced memory cost for aggregation by moving from in-memory to the on-disk LSM based approach.

Reply:
I wanted to do so, but struggled to word it properly :facepalm:

Reply:
Updated